src/typing/constraint_js.ml

```diff
@@ -2167,6 +2167,40 @@ let reasons_of_trace ?(level=0) ?(tab=2) trace =
     else []
   in f level trace
 
+
```

Comment:
Are you sure this PR is properly rebased on top of master? These changes shouldn't be showing up here since they happened in another commit...